cinema/views.py

```diff
@@ -167,6 +219,46 @@ def get_serializer_class(self):
 
         return MovieSessionSerializer
 
+    @extend_schema(
+        parameters=[
+            OpenApiParameter(
+                "date",
+                type={"type": "string"},
+                description="Filter movie sessions by day when show starts("
+                "ex. "
+                "?date=%Y-%m-%d)",
+            ),
+            OpenApiParameter(
+                "movie",
+                type={"type": "integer"},
+                description="Filter by id's genres(ex. ?movie=2)",
+            ),
+        ]
+    )
+    def list(self, request, *args, **kwargs):
+        """Endpoint for listing movie_sessions"""
+        return super().list(request, *args, **kwargs)
+
+    def create(self, request, *args, **kwargs):
+        """endpoint for creating movie_sessions"""
+        return super().create(request, *args, **kwargs)
+
+    def retrieve(self, request, *args, **kwargs):
+        """endpoint for retrieving movie_sessions"""
+        return super().retrieve(request, *args, **kwargs)
+
+    def update(self, request, *args, **kwargs):
+        """endpoint for updating movie_sessions"""
+        return super().update(request, *args, **kwargs)
+
+    def partial_update(self, request, *args, **kwargs):
+        """endpoint for patchong movie_sessions"""
```

Comment:
There is a typo in the docstring for the `partial_update` method. It should be 'endpoint for patching movie_sessions' instead of 'endpoint for patchong movie_sessions'.